docs/content/preview/contribute/core-database/clion-setup.md

```diff
@@ -51,3 +51,23 @@ YB_USE_NINJA=0 ./yb_build.sh
 ```
 
 Subsequent builds can be launched also from CLion.
+
+### Configure as a compilation database project
+
+To improve performance, you can try opening the project as a compilation database project:
+
+1. Run `./yb_build.sh compilecmds` to generate the `compile_commands.json` file in the `build` directory.
```

Comment:
```suggestion
1. Run `./yb_build.sh compilecmds` to generate the `compile_commands.json` file in the `yugabyte-db` directory.
```




2. Verify that `compile_commands.json` is present in the `yugabyte-db` folder. From the `yugabyte-db` folder. You should see something like this:
```
$ find . -name "compile_commands.json"
./compile_commands.json
./build/compilecmds-clang-dynamic-arm64-ninja/compile_commands.json
./build/compilecmds-clang-dynamic-arm64-ninja/compile_commands/combined_raw/compile_commands.json
./build/compilecmds-clang-dynamic-arm64-ninja/compile_commands/yb_postprocessed/compile_commands.json
./build/compilecmds-clang-dynamic-arm64-ninja/compile_commands/yb_raw/compile_commands.json
./build/compilecmds-clang-dynamic-arm64-ninja/compile_commands/combined_postprocessed/compile_commands.json
./build/compilecmds-clang-dynamic-arm64-ninja/compile_commands/pg_raw/compile_commands.json
./build/compilecmds-clang-dynamic-arm64-ninja/compile_commands/pg_postprocessed/compile_commands.json
./build/compilecmds-clang-dynamic-arm64-ninja/postgres_build/compile_commands.json
./build/compilecmds-clang-dynamic-arm64-ninja/postgres_build/contrib/compile_commands.json
./build/compilecmds-clang-dynamic-arm64-ninja/postgres_build/third-party-extensions/compile_commands.json
```
If `./compile_commands.json` is not there, then make a symlink to the one in the `build` folder:

```
ln -s build/compilecmds-clang-dynamic-arm64-ninja/compile_commands/combined_postprocessed/compile_commands.json compile_commands.json
```
Replace `compilecmds-clang-dynamic-arm64-ninja` as needed.

Reply:
User comment said the json was created in build directory?